src/memgraph.cpp

```diff
@@ -669,6 +669,9 @@ int main(int argc, char **argv) {
     websocket_server.Shutdown();
 #ifdef MG_ENTERPRISE
     metrics_server.Shutdown();
+    if (coordinator_state.has_value() && coordinator_state->IsCoordinator()) {
+      coordinator_state->ShutDownCoordinator();
+    }
```

Comment:
Why are we doing this?

Reply:
Coordinator might be in infinite loop doing ForceReset and it might be stuck and we might not shut down it at any point soon. So we are doing shutdown this way

Reply:
I will write on force reset why are we doing this